<!DOCTYPE html>



  


<html class="theme-next gemini use-motion" lang="">
<head>
  <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
<meta name="theme-color" content="#222">



  
  
    
    
  <script src="/lib/pace/pace.min.js?v=1.0.2"></script>
  <link href="/lib/pace/pace-theme-flash.min.css?v=1.0.2" rel="stylesheet">







<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />
















  
  
  <link href="/lib/fancybox/source/jquery.fancybox.css?v=2.1.5" rel="stylesheet" type="text/css" />







<link href="/lib/font-awesome/css/font-awesome.min.css?v=4.6.2" rel="stylesheet" type="text/css" />

<link href="/css/main.css?v=5.1.4" rel="stylesheet" type="text/css" />


  <link rel="apple-touch-icon" sizes="180x180" href="/images/favicon.png?v=5.1.4">


  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon.png?v=5.1.4">


  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon.png?v=5.1.4">


  <link rel="mask-icon" href="/images/favicon.png?v=5.1.4" color="#222">





  <meta name="keywords" content="Neuroscience," />





  <link rel="alternate" href="/atom.xml" title="La Sobremesa" type="application/atom+xml" />






<meta name="description" content="This essay was for the midterm examination of curriculum Introduction to Neuroscience when I was in my junior year of university. It took me about one month to read over 50 papers, and two weeks to dr">
<meta name="keywords" content="Neuroscience">
<meta property="og:type" content="article">
<meta property="og:title" content="Connectome: From Structure to Function">
<meta property="og:url" content="http://lacroqueta.me/2018/09/13/Connectome/index.html">
<meta property="og:site_name" content="La Sobremesa">
<meta property="og:description" content="This essay was for the midterm examination of curriculum Introduction to Neuroscience when I was in my junior year of university. It took me about one month to read over 50 papers, and two weeks to dr">
<meta property="og:locale" content="default">
<meta property="og:image" content="http://lacroqueta.me/2018/09/13/Connectome/connectome.png">
<meta property="og:updated_time" content="2018-10-10T06:49:20.596Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Connectome: From Structure to Function">
<meta name="twitter:description" content="This essay was for the midterm examination of curriculum Introduction to Neuroscience when I was in my junior year of university. It took me about one month to read over 50 papers, and two weeks to dr">
<meta name="twitter:image" content="http://lacroqueta.me/2018/09/13/Connectome/connectome.png">



<script type="text/javascript" id="hexo.configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Gemini',
    version: '5.1.4',
    sidebar: {"position":"right","display":"post","offset":12,"b2t":false,"scrollpercent":false,"onmobile":false},
    fancybox: true,
    tabs: true,
    motion: {"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},
    duoshuo: {
      userId: '0',
      author: 'Author'
    },
    algolia: {
      applicationID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    }
  };
</script>



  <link rel="canonical" href="http://lacroqueta.me/2018/09/13/Connectome/"/>





  <title>Connectome: From Structure to Function | La Sobremesa</title>
  




<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
          m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-127164688-1', 'auto');
  ga('send', 'pageview');
</script>





</head>

<body itemscope itemtype="http://schema.org/WebPage" lang="default">

  
  
    
  

  <div class="container sidebar-position-right page-post-detail">
    <div class="headband"></div>

    <header id="header" class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-wrapper">
  <div class="site-meta ">
    

    <div class="custom-logo-site-title">
      <a href="/"  class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">La Sobremesa</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
      
        <p class="site-subtitle"></p>
      
  </div>

  <div class="site-nav-toggle">
    <button>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
    </button>
  </div>
</div>

<nav class="site-nav">
  

  
    <ul id="menu" class="menu">
      
        
        <li class="menu-item menu-item-home">
          <a href="/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-home"></i> <br />
            
            Home
          </a>
        </li>
      
        
        <li class="menu-item menu-item-tags">
          <a href="/tags/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-tags"></i> <br />
            
            Tags
          </a>
        </li>
      
        
        <li class="menu-item menu-item-archives">
          <a href="/archives/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-archive"></i> <br />
            
            Archives
          </a>
        </li>
      

      
    </ul>
  

  
</nav>



 </div>
    </header>

    <main id="main" class="main">
      <div class="main-inner">
        <div class="content-wrap">
          <div id="content" class="content">
            

  <div id="posts" class="posts-expand">
    

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://lacroqueta.me/2018/09/13/Connectome/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="Jason Zheng">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="La Sobremesa">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">Connectome: From Structure to Function</h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-09-13T22:37:49+08:00">
                2018-09-13
              </time>
            

            

            
          </span>

          

          
            
              <span class="post-comments-count">
                <span class="post-meta-divider">|</span>
                <span class="post-meta-item-icon">
                  <i class="fa fa-comment-o"></i>
                </span>
                <a href="/2018/09/13/Connectome/#comments" itemprop="discussionUrl">
                  <span class="post-comments-count valine-comment-count" data-xid="/2018/09/13/Connectome/" itemprop="commentCount"></span>
                </a>
              </span>
            
          

          
          

          

          
            <div class="post-wordcount">
              
                
                <span class="post-meta-item-icon">
                  <i class="fa fa-file-word-o"></i>
                </span>
                
                  <span class="post-meta-item-text">Words count in article&#58;</span>
                
                <span title="Words count in article">
                  5.2k
                </span>
              

              
                <span class="post-meta-divider">|</span>
              

              
                <span class="post-meta-item-icon">
                  <i class="fa fa-clock-o"></i>
                </span>
                
                  <span class="post-meta-item-text">Reading time &asymp;</span>
                
                <span title="Reading time">
                  32
                </span>
              
            </div>
          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        <p>This essay was for the midterm examination of curriculum <em>Introduction to Neuroscience</em> when I was in my junior year of university. It took me about one month to read over 50 papers, and two weeks to draft and to complete. It is still easy to read and understand, in my opinion, and scientific enough to give you the overview of the connectome. Hope you all enjoy it.</p>
<p><img src="/2018/09/13/Connectome/connectome.png" alt="&quot;test&quot;"></p>
<center>From <a href="http://www.humanconnectomeproject.org/" target="_blank" rel="noopener">Human Connectome Project</a></center>

<h2 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h2><p>Our brain is a complicated network which is unified for structural and functional system. However, currently our knowledge is fairly scarce, while it is essential to learn our brain for the progress of humanity. This article will introduce the concept, meanwhile also an undergoing project, called the connectome. The background and history of connectome are set out first, from the time when Cajal opened the new door of brain science, to one decade ago when this concept was put forward to the public. Then three scales are briefly introduced which is divided to do the research more efficiently, on the basis of the structure and imaging resolution. In addition, the techniques and approaches utilized and the tendency they may development towards are illustrated as well. Finally,  a general conclusion on connectome is made, including the current applications and bright prospects which is to be seen recently and in the future.</p>
<a id="more"></a>
<h2 id="Introduction"><a href="#Introduction" class="headerlink" title="Introduction"></a>Introduction</h2><p>Networks are everywhere nowadays. When we pick up a fallen leaf, we can see the veins on it, which is the network under the leaf surface. When we go out of our house, facing toward our destination, there is a big network consisting of roads and intersections in front of us. And of course, the Internet, which we are both familiar and strange with. For most of us, we use it every day, without knowing what the certain principles is inside. Therefore, we may get kind of helpless, even a little bit desperate, when something is wrong with the Internet, with nobody owing professional knowledge around. For human being, Internet, once created by our own hand, is so complicated that we struggled to learn to learn it. Let alone those networks that were masterpieces created by the nature and is still largely unknown to us. </p>
<p>In our brain, there exists a mysterious network. As we all know, the most functional unit of our brain is one type of cell, called neurons. There are millions of neurons in human’s brain, making it obvious that some certain interconnections and interactions must exist between the massive individuals. One neuron is composed of soma, axons and dendrites. The information transports from one neuron to another through synapses, which are formed by axons from one neuron and dendrites from another. Several neurons collaborate, to function as a circuit. Several circuits collaborate, to function as a sub-area of the brain. Several sub-areas collaborate, to make our brains work. </p>
<p>The term “Connectome” was first seen in 2005, suggested by Dr. Olaf Sporns at Indiana University and Dr. Patric Hagmann at Lausanne University Hospital. The connectome is a comprehensive map of neural connections in the brain. As Dr. Olaf Sporns wrote in his paper, “The connectome will significantly increase our understanding of how functional brain states emerge from their underlying structural substrate and will provide new mechanistic insights into how brain function is affected if this structural substrate is disrupted. It will provide a unified, time-invariant, and readily available neuroinformatic resources that could be used in virtually all areas of experimental and theoretical neuroscience.’’<sup>1</sup></p>
<p>In fact, the term is inspired by genome, another ongoing project with periodic fruit. The two subjects are indeed comparable. Dr. Patric Hagmann pointed out the similarities between genome and connectome in his paper, “It is clear that, like the genome, which is much more than just a juxtaposition of genes, the set of all neuronal connections in the brain is much more than the sum of their individual components. The genome is an entity itself, as it is from the subtle gene interaction that life emerges. In a similar manner, one could consider the brain connectome, set of all neuronal connections, as one single entity, thus emphasizing the fact that the huge brain neuronal communication capacity and computational power critically relies on this subtle and incredibly complex connectivity architecture.”<sup>2</sup></p>
<p>As is mentioned above, there are quite many similarities between connectome and genome. At first, they are both fundamental structural data sets with regard to the unit they function. Also, there are many levels in the two concepts, especially according to different scales, with synapse, neurons, circuits and sub-region in connectome, while with nucleotide, histones, genes, and chromosome in genome. Thirdly, both of them are different across different species, or different individuals of the same species, or the same species from different period in the long river of time. In addition, although in one individual, they are variable, probably due to environment, requirements for different functions or the spatial proximity.<sup>3-5</sup> At last, complex system dynamics are produced, for instance, the firing patterns of neurons and the spatiotemporal expression of genes.<sup>6</sup></p>
<p>Just like the time when the genome project first came up, there are many objections and worries against the connectome project, such as insight limitation, poor using of scarce source, seeming impossible task and so on.<sup>7</sup> Anyway so far there have been some achievements towards the goal. About 30 years ago, a work which was consistent with the connectome virtually have been accomplished. The structure of the nervous system of <em>C. elegans</em> was fully constructed.<sup>8-10</sup> There are exactly 302 neurons, without varying structures in <em>C. elegans</em>, making it convenient for study. Although the structure of the nervous system of nematode is much simpler than other species, especially the mammals, the complete data set is still too huge to be handled. Furthermore, partial connectomes of a mouse retina and that of a mouse primary visual cortex have been reconstructed.<sup>11,12</sup> What’s more, in 2014, The Allen Mouse Brain Connectivity Atlas presented a mesoscale connectivity of the adult mouse brain, which is available as an open-access public database.<sup>13</sup></p>
<p>The ultimate goal is to reconstruct human connectome, which is, “a comprehensive structural description of the network of elements and connections forming the human brain.”<sup>1,2</sup> There is no doubt that the project is significant for us human being. Only if we know what the structure of our brain is like, can we say it is possible to learn the unambiguous function. Otherwise, everything we are doing is just like blind monks touching the elephant to try to depict it, without any directions or strategies or efficiency. There are many groups devoting themselves to explore the secret of human connectome, while two of them in the lead, both of which were awarded Human Connectome Project (HCP) granted by NIH in 2010. One is led by Washington University, the University of Minnesota, and Oxford University, called the “Wu-Minn” HCP consortium, and the other is led by Massachusetts General Hospital and University of California at Los Angeles, called the MGH-UCLA HCP consortium.<sup>14</sup></p>
<h2 id="Scales-and-Levels"><a href="#Scales-and-Levels" class="headerlink" title="Scales and Levels"></a>Scales and Levels</h2><p>As is similar to genome, which can be studied in different scales of base pairs, nucleotides, genes, chromosomes and so on, there are different scales to watch the connectome. Generally speaking, these scales can be categorized as microscale, mesoscale, and macroscale, intelligible as the name suggests, defined according to different levels of spatial resolution in brain imaging. Different scales provide different fundamental units and elements to study, different methods to utilize, different challenges to face, and different results to get in the end. As a result of that it is quite significant for us to learn about what the certain scales are, and what information they can provide. </p>
<h3 id="Microscale"><a href="#Microscale" class="headerlink" title="Microscale"></a>Microscale</h3><p>It is the most exquisite scale among the three mentioned above, which means the units and elements we are facing is the most fundamental. It is a world between neurons and synapses, just like what we see under the electron microscope. In microscale, we need to know every interaction between every neuron. It is not as simple as one-to-one correspondence. As we all know, the axon of one neuron forms synapses with the spines of the dendrites on another neuron. The problem is, there are many spines on one dendrite, and there are many dendrites on one neuron. Besides, there can be multiple spots which is capable for forming synapse on one axon terminate, as is suggested in rodent mesoaccumbens axons.<sup>15</sup></p>
<p>The consequence is that there are about 10<sup>10</sup> neurons in human cerebral cortex, and 10<sup>14</sup> synaptic connections between them.<sup>16</sup> For now, there is no efficient techniques and to do the mass data collection, and we also have no reliable method or theory to do the data analyzing. In addition, the synaptic structures are variable though different times,<sup>3,17</sup> which lifts the dimensions of the problem itself and the difficulties for multiple levels. It seems that the variable remodeling of spines may have an influence on the synaptic connections of abundant potential synaptic sites.<sup>1,18,19</sup> In conclusion, apart from the difficulties we may come across while studying connectome in the microscale, the necessity and effectiveness are also worth questioning. </p>
<h3 id="Macroscale"><a href="#Macroscale" class="headerlink" title="Macroscale"></a>Macroscale</h3><p>According to the normal logic, the order in which I introduce the three scales is from the smallest to the biggest, or the reverse. However, once upon I finished the microscale part, preparing for the mesoscale part in details, I found the concept of mesoscale is ambiguous, while that of macroscale is much clearer. It is like the two sides are almost decided, so there is much space for the middle to adjust itself. As a result, I decided to change my order, putting the macroscale part in front of the mesoscale part.</p>
<p>Although the neurons have their own significance in the whole system, some of them have very common points that make them work collaboratively, implementing different functions. If we recognize these similar neurons as a whole part, the brain can be briefly divided into several regions, with their own significance of existing and function. The earliest classification of brain regions is developed by Franz Joseph in the nineteenth century. He came up with a discipline called phrenology, divided the brain into several regions marking different mental functions, like ideality, hope and consciousness.<sup>20</sup> As our knowledge about brain gets deeper, the division of brain regions have been modified for many times. The basis of the division of the brain region is that all the “structural elements share highly similar long-range connectivity patterns”, and that “these patterns are dissimilar between brain regions”.<sup>1</sup> Anyway, no matter how the regions change, they are the nodes that make up the brain graph in macroscale. </p>
<h3 id="Mesoscale"><a href="#Mesoscale" class="headerlink" title="Mesoscale"></a>Mesoscale</h3><p>As is mentioned above, the definition of mesoscale is quite ambiguous, between those of microscale and macroscale. However, many scientists prefer the word “neuron population” to define the tricky concept.<sup>1,6,13,21</sup> It is under a spatial resolution of hundreds of micrometers, containing hundreds or thousands of individual neurons. According to Dr. Olaf Sporns, the definition of cortical minicolumn is an appropriate example.<sup>1</sup> The dimeter of one minicolumn is about 30-50 micrometers, containing approximately 80-100 individual neurons.<sup>22</sup> As the same as the rule of macroscale, it is also required to be highly similar with the others in one area, and to show difference compared to external properties. </p>
<p>It is no doubt that the mesoscale and macroscale view are the best two for connectome studying. And in fact, there has been some effectual practice in these two scales. A mesoscale connectome of the mouse brain has been reconstructed by Allen Institute for Brain Science,<sup>13</sup> and was further later.<sup>23</sup> The macroscale connectivity are also be visualized in mouse<sup>24</sup> and human brain.<sup>25</sup> It is believed that the combination of data of connectivity in micro-, meso-, and macroscale will be complementary to each other, making it more convenient, comprehensive, and deep for us to understand the connectome. </p>
<h2 id="Techs-and-Methods"><a href="#Techs-and-Methods" class="headerlink" title="Techs and Methods"></a>Techs and Methods</h2><p>In the recent one century, scientists who are studying the connectivity of neurons in the brain have used three main methods, the single-cell impregnation, the optically based tract-tracing and electron microscopy.<sup>26</sup> These methods have helped us solve many problems and make the great discoveries, yet, may have more and more drawbacks and restrictions as the technology and our work progresses. In this part, I will introduce some current methods, including but not limited to those mentioned above, and some ideal ones.</p>
<p>The single-cell staining by dye impregnation is the very first approach to study the brain. It can be traced back to the nineteenth century, when Cajal began to watch the nervous system using Golgi staining. It is known what happened next, and where are standing is based on the great discovery and the great approach. Nevertheless, in spite of its contribution, staining, whether the past or current method, has still not solved some problems. Firstly, because of the light diffraction, there cannot be a high resolution, which makes it impossible to discern in a subtle degree. Secondly, the distance that the chemicals can travel is limited, making it too hard to see the overall situation to make an accurate judgement.</p>
<p>To make the long-distance observation feasible, scientists then used tract-tracing techniques, in which surgeries were usually required. Afterwards, it was replaced by techniques in which labels was put in certain areas and then diffused to the target regions. There is a wide range of labels, such as radioactive amino acids or sugars, proteins, enzymes and fluorescent organic molecules.<sup>27</sup> As the genetic approaches developments, labels like fluorescent proteins can be encoded in certain times or regions to express, under the control of regulatory elements or viral vectors.<sup>26,28-32</sup> However, a coin has its two sides. Tract-tracing techniques can only trace large neuron populations and single neural pathways, which makes it not so useful if not combining with other complementary approaches.</p>
<p>For another, electron microscopy can resolve the resolution problem, for it is the most observing techniques nowadays. Electron microscopy played an important role in the construction of connectome of <em>C. elegans</em> in 1986.<sup>8,33</sup> For now, there are many modified combinational methods on the basis of observing via the electron microscopy. Just like 3D print, we can put the tissue into very thin slices, watching and analyzing the information on each individual slice, finally reconstructing them. A new technique utilizing a tape-like structure, makes the cutting and collecting consecutive and automated.<sup>34</sup> Nevertheless, more delicate, more work. Using the slices, which is full of detailed information, means unimaginable information to collect, analyze and integrate. At least currently, it seems lack of hope to do the huge work with techniques we master, let alone by hand. </p>
<p>In addition, there is also a problem that the construction of the nerve system requires explicit identification of each neurons we are interested. The process cannot be done if there is only one or two single labels being used. Brainbow is such an approach that aiming at this problem. Just as its name, which is inspired by rainbow, a beautiful seven-color natural phenomenon, the function if brainbow is to label individual neurons in stochastic colors, via the expression of multiple fluorescent proteins.<sup>35</sup> The Human Connectome Project even makes a gallery to exhibit the gorgeous artwork-like pictures which shows the structure of the white matter in colors. However, as many other approaches, brainbow has its own limitations. For instance, the diffraction-limited resolution is still a problem. Also, the effect of color display needs to be improved further, such as the brightness, distinguishing degree. What’s more, the expression in projection neurons are high, while that in interneurons are poor.<sup>36</sup></p>
<p>Inspired by brainbow, here came a novel approach called BOINC, which stands for barcoding of individual neuronal connections.<sup>37</sup> This approach utilizes the convenience and speediness of DNA sequencing. Each neuron is labeled by a unique sequence of nucleotides, just like a barcode. And these barcodes from neurons which are synaptically connected are associated together. As a result, the two types of barcodes, representing one neuron in a sense, is paired, and then be sequenced of high throughput. It is considered to be potential to be cheap and fast.</p>
<p>It seems there is a lot of work that has been and is being done. However, those works are on the basis of static state. Imaging is not a gif-format animation, or a movie. It can only capture a stop motion for a very short-time moment. For another reason, currently most of the imaging works are conducted post mortem, which means we cannot adjust our experiments for further study, on the same individual, much less real-time monitoring. Some non-invasive imaging techniques provide new approaches about solving the problem.</p>
<p>MRI, the abbreviation of magnetic resonance imaging, has been a traditional and common approach for imaging the modality of brain.<sup>38</sup> It doesn’t use ionizing radiation, and the data it acquired can be applied for both clinical and research subjects. Repeated scanning of the same individual is also viable. However, the drawback is a long-time stationary state is required, which is difficult for animals. Anyway, MRI provides a basic tech platform that many other techniques are born from it, like dMRI, standing for diffusion MRI, and fMRI, standing for functional MRI.<sup>39</sup> The first provides cubic-millimeter-resolution construction of white-matter tracts,<sup>25</sup> while the second makes it feasible to capture the activity of the brain, revealing a universal functional architecture.<sup>40</sup></p>
<p>As we acquire structural and functional data, it is time to combine them with some appropriate logic and methods. Four steps are needed to achieve this goal.<sup>41</sup> Firstly, it is essential to determine the parcellation of the brain, and then obtain the data for both structural and functional aiming approaches. Secondly, express the structural linkage and functional coupling. Thirdly, build a connection matrix aggregating both the structural and functional network. Lastly, network analysis can be started by computing and extracting relevant information from the two matrix.<sup>6</sup></p>
<p>It is obviously not enough to achieve human connectome. The follow-up work was suggested by Dr. Olaf Sporns in his milestone-like paper in 2005.<sup>1</sup> After obtaining the data from network analysis, it is required to test and verify the human connectome. We can perform population analysis of healthy subjects and spatial registration to adjust the result acquired by all of computing to the reality. In addition, we can compare connectome of healthy human with that of patients or compare connectome of human being with that of other species evolutionally next to us, like the macaque. There is inestimable information and value waiting to be discovered.</p>
<h2 id="Challenges"><a href="#Challenges" class="headerlink" title="Challenges"></a>Challenges</h2><h3 id="Complexity"><a href="#Complexity" class="headerlink" title="Complexity"></a>Complexity</h3><p>As is through the whole article, the reason we spent decades and decades, centuries and centuries, on observing, depicting, dividing and analyzing the brain, and the reason we have and still being developed techniques and approaches one after another, and the reason we are making new discoveries, overthrowing and pruning the previous view, is not only the brain is so important to us, but also it is so complex and variable. The complexity has been involved above but is far more than that. For any levels in the system, ranging from neurons to brain region, is much more complicated than we could imagine. For instance, we are devoting to finding the connections between the structure and function. However, there is a large number of functional connections between regions that are not directedly connect, in anatomical view.<sup>42</sup> The complexity increases the difficulty for us to observe the structure, dispose the data, look for the latent connections, select the appropriate approach, learn and understand the function. </p>
<h3 id="Individual-Variability"><a href="#Individual-Variability" class="headerlink" title="Individual Variability"></a>Individual Variability</h3><p>Just like the genome, connectome represents a relatively common characteristic of one specie. However, as to individuals, they can have their own connectome. As we all know, everybody of us are different from others’, for we have our own DNA, which carries our own genetic material. The DNA determines what our connectome is like, so our own connectome is different from others’. Apart from genetic differences, there are many other causes resulting in individual connectome variable, such as developmental and experiential history, gender difference, pathologies and responses to injury.<sup>1</sup> If connectome is anticipated to make contributions to certain areas, such as diseases, the individual variability cannot be neglected. </p>
<h3 id="Parcellations-and-Scales"><a href="#Parcellations-and-Scales" class="headerlink" title="Parcellations and Scales"></a>Parcellations and Scales</h3><p>Another challenge is the criterion of parcellations and scales of the brain. Although there are three recognized relatively clear scales, a certain level of resolution to complete the connectome is still under the debate. With current techniques, we may begin the project from mesoscale and macroscale. It can be foreseen that as we learn more about the nerve system, especially the tinier functional structure than synapses, or even, transmitters, and as we invent more powerful machines and tools, there seems to be an endless processing. And the parcellation also matters, for whether the accurate boundaries of different regions can be found. The lack of knowledge to our brain is the restrictions to achieve the goal, which needs more work along with the whole brain science.</p>
<h3 id="Techs"><a href="#Techs" class="headerlink" title="Techs"></a>Techs</h3><p>In essence, the lack of knowledge is due to the undeveloped technology. Currently, the acquisition, storage and manipulation of raw data is still a headache problem. It is estimated that a single microscale human connectome may take up one trillion gigabytes.<sup>43</sup> There are also a large number of choke points on our way leading to the truth. As to connectome, there are majorly two types of techniques that are badly required. One is imaging, which can make it clear at a very high resolution, identifying different components at the same time, and the other is analyzing, making it automated and highly efficient to processing abundant data set. Looking back on the history of neuroscience, or a bit wider, the bioscience, each breakthrough made to the field and for human, always came after a new revolutionary technique. The only way to make a breakthrough is the invention of new techniques. Particularly outstanding discoveries and outcomes cannot be made if the technology doesn’t make any progress. </p>
<h2 id="Outcome"><a href="#Outcome" class="headerlink" title="Outcome"></a>Outcome</h2><p>The direction from basic research for brain science to connectome is no doubt correct. Brain plays such an important role that if the relationship between its structure and function is figured out, imagine what great changes will be brought to our lives! Our intelligence may improve, and we may even amend the personalities that we do not want. We can even get to know what happens in our brain when we are retrieving our certain memory. If you have seen the famous science fiction TV serious <em>Black Mirror</em>, you may not be strange to the scene that the roles in one episode can reviewing and deleting their memory, as seems fantastic but possibly come true someday in the future. Moreover, some serious diseases, like the Parkinson’s disease and the tristimania, may be cured. Several progresses have been made through recent years such as the disorder connectivities of brain in Schizophrenia,<sup>44</sup> and aging.<sup>45</sup> It is also possible for disabled people who have lost their fingers or hands to regain the ability to seize tiny things. The potential of connectome is difficult to estimate and imagine, and so is the profit that can be brought to us. What we are waiting for is a revolutionary researching system, or just an approach, to help us integrating and analyzing the massive raw data. The day when comes the revolution, is the day when we decipher our brain, and is the day when we can change the world.</p>
<h2 id="Reference"><a href="#Reference" class="headerlink" title="Reference"></a>Reference</h2><p>1          Sporns, O., Tononi, G. &amp; Kotter, R. The human connectome: A structural description of the human brain. <em>PLoS Comput Biol</em> <strong>1</strong>, e42, doi:10.1371/journal.pcbi.0010042 (2005).</p>
<p>2          Hagmann, P. <em>From diffusion MRI to brain connectomics</em>, Institut de traitement des signaux PROGRAMME DOCTORAL EN INFORMATIQUE ET COMMUNICATIONS POUR L’OBTENTION DU GRADE DE DOCTEUR ÈS SCIENCES PAR Docteur en médecine, Université de Lausanne, (2005).</p>
<p>3          Bonhoeffer, T. &amp; Yuste, R. Spine motility. Phenomenology, mechanisms, and function. <em>Neuron</em> <strong>35</strong>, 1019-1027 (2002).</p>
<p>4          Fraga, M. F. <em>et al.</em> Epigenetic differences arise during the lifetime of monozygotic twins. <em>Proc Natl Acad Sci U S A</em> <strong>102</strong>, 10604-10609, doi:10.1073/pnas.0500398102 (2005).</p>
<p>5          Rajapakse, I., Scalzo, D., Tapscott, S. J., Kosak, S. T. &amp; Groudine, M. Networking the nucleus. <em>Mol Syst Biol</em> <strong>6</strong>, 395, doi:10.1038/msb.2010.48 (2010).</p>
<p>6          Sporns, O. The human connectome: a complex network. <em>Ann N Y Acad Sci</em> <strong>1224</strong>, 109-125, doi:10.1111/j.1749-6632.2010.05888.x (2011).</p>
<p>7          Lichtman, J. W. &amp; Sanes, J. R. Ome sweet ome: what can the genome tell us about the connectome? <em>Curr Opin Neurobiol</em> <strong>18</strong>, 346-353, doi:10.1016/j.conb.2008.08.010 (2008).</p>
<p>8          White, J. G., Southgate, E., Thomson, J. N. &amp; Brenner, S. The structure of the nervous system of the nematode Caenorhabditis elegans. <em>Philos Trans R Soc Lond B Biol Sci</em> <strong>314</strong>, 1-340 (1986).</p>
<p>9          Chen, B. L.-J. <em>Neuronal network of C. elegans: from anatomy to behavior</em>, Cold Spring Harbor Laboratory, (2007).</p>
<p>10        Varshney, L. R., Chen, B. L., Paniagua, E., Hall, D. H. &amp; Chklovskii, D. B. Structural properties of the Caenorhabditis elegans neuronal network. <em>PLoS Comput Biol</em> <strong>7</strong>, e1001066 (2011).</p>
<p>11        Briggman, K. L., Helmstaedter, M. &amp; Denk, W. Wiring specificity in the direction-selectivity circuit of the retina. <em>Nature</em> <strong>471</strong>, 183-188, doi:10.1038/nature09818 (2011).</p>
<p>12        Bock, D. D. <em>et al.</em> Network anatomy and in vivo physiology of visual cortical neurons. <em>Nature</em> <strong>471</strong>, 177-182, doi:10.1038/nature09802 (2011).</p>
<p>13        Oh, S. W. <em>et al.</em> A mesoscale connectome of the mouse brain. <em>Nature</em> <strong>508</strong>, 207-214, doi:10.1038/nature13186 (2014).</p>
<p>14        Van Essen, D. C. <em>et al.</em> The WU-Minn Human Connectome Project: an overview. <em>Neuroimage</em> <strong>80</strong>, 62-79, doi:10.1016/j.neuroimage.2013.05.041 (2013).</p>
<p>15        Zhang, S. <em>et al.</em> Dopaminergic and glutamatergic microdomains in a subset of rodent mesoaccumbens axons. <em>Nat Neurosci</em> <strong>18</strong>, 386-392, doi:10.1038/nn.3945 (2015).</p>
<p>16        Young, M. P. The organization of neural systems in the primate cerebral cortex. <em>Proc Biol Sci</em> <strong>252</strong>, 13-18, doi:10.1098/rspb.1993.0040 (1993).</p>
<p>17        Engert, F. &amp; Bonhoeffer, T. Dendritic spine changes associated with hippocampal long-term synaptic plasticity. <em>Nature</em> <strong>399</strong>, 66-70, doi:10.1038/19978 (1999).</p>
<p>18        Stepanyants, A., Hof, P. R. &amp; Chklovskii, D. B. Geometry and structural plasticity of synaptic connectivity. <em>Neuron</em> <strong>34</strong>, 275-288 (2002).</p>
<p>19        Kalisman, N., Silberberg, G. &amp; Markram, H. The neocortical microcircuit as a tabula rasa. <em>P Natl Acad Sci USA</em> <strong>102</strong>, 880-885, doi:10.1073/pnas.0407088102 (2005).</p>
<p>20        Luo, L. <em>Principles of Neurobiology</em>.  (Garland Science, 2015).</p>
<p>21        Rah, J. C., Feng, L., Druckmann, S., Lee, H. &amp; Kim, J. From a meso- to micro-scale connectome: array tomography and mGRASP. <em>Front Neuroanat</em> <strong>9</strong>, 78, doi:10.3389/fnana.2015.00078 (2015).</p>
<p>22        Mountcastle, V. B. The columnar organization of the neocortex. <em>Brain</em> <strong>120</strong>, 701-722, doi:DOI 10.1093/brain/120.4.701 (1997).</p>
<p>23        Henriksen, S., Pang, R. &amp; Wronkiewicz, M. A simple generative model of the mouse mesoscale connectome. <em>Elife</em> <strong>5</strong>, doi:ARTN e12366</p>
<p>10.7554/eLife.12366 (2016).</p>
<p>24        Stafford, J. M. <em>et al.</em> Large-scale topology and the default mode network in the mouse connectome. <em>P Natl Acad Sci USA</em> <strong>111</strong>, 18745-18750, doi:10.1073/pnas.1404346111 (2014).</p>
<p>25        Craddock, R. C. <em>et al.</em> Imaging human connectomes at the macroscale. <em>Nature Methods</em> <strong>10</strong>, 524-539, doi:10.1038/Nmeth.2482 (2013).</p>
<p>26        Lichtman, J. W., Livet, J. &amp; Sanes, J. R. A technicolour approach to the connectome. <em>Nat Rev Neurosci</em> <strong>9</strong>, 417-422, doi:10.1038/nrn2391 (2008).</p>
<p>27        Callahan, C. A., Yoshikawa, S. &amp; Thomas, J. B. Tracing axons. <em>Curr Opin Neurobiol</em> <strong>8</strong>, 582-586 (1998).</p>
<p>28        Feinstein, P., Bozza, T., Rodriguez, I., Vassalli, A. &amp; Mombaerts, P. Axon guidance of mouse olfactory sensory neurons by odorant receptors and the beta 2 adrenergic receptor. <em>Cell</em> <strong>117</strong>, 833-846, doi:DOI 10.1016/j.cell.2004.05.013 (2004).</p>
<p>29        Bareyre, F. M., Kerschensteiner, M., Misgeld, T. &amp; Sanes, J. R. Transgenic labeling of the corticospinal tract for monitoring axonal responses to spinal cord injury. <em>Nat Med</em> <strong>11</strong>, 1355-1360, doi:10.1038/nm1331 (2005).</p>
<p>30        Dymecki, S. M. &amp; Kim, J. C. Molecular neuroanatomy’s “three Gs”: A primer. <em>Neuron</em> <strong>54</strong>, 17-34, doi:10.1016/j.neuron.2007.03.009 (2007).</p>
<p>31        Song, C. K., Enquist, L. W. &amp; Bartness, T. J. New developments in tracing neural circuits with herpesviruses. <em>Virus Res</em> <strong>111</strong>, 235-249, doi:10.1016/j.virusres.2005.04.012 (2005).</p>
<p>32        Luo, L., Callaway, E. M. &amp; Svoboda, K. Genetic dissection of neural circuits. <em>Neuron</em> <strong>57</strong>, 634-660, doi:10.1016/j.neuron.2008.01.002 (2008).</p>
<p>33        Macagno, E. R., Levinthal, C. &amp; Sobel, I. Three-dimensional computer reconstruction of neurons and neuronal assemblies. <em>Annu Rev Biophys Bioeng</em> <strong>8</strong>, 323-351, doi:10.1146/annurev.bb.08.060179.001543 (1979).</p>
<p>34        Kasthuri, N. <em>et al.</em> Saturated Reconstruction of a Volume of Neocortex. <em>Cell</em> <strong>162</strong>, 648-661, doi:10.1016/j.cell.2015.06.054 (2015).</p>
<p>35        Livet, J. <em>et al.</em> Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system. <em>Nature</em> <strong>450</strong>, 56-62, doi:10.1038/nature06293 (2007).</p>
<p>36        Feng, G. <em>et al.</em> Imaging neuronal subsets in transgenic mice expressing multiple spectral variants of GFP. <em>Neuron</em> <strong>28</strong>, 41-51 (2000).</p>
<p>37        Zador, A. M. <em>et al.</em> Sequencing the connectome. <em>PLoS Biol</em> <strong>10</strong>, e1001411, doi:10.1371/journal.pbio.1001411 (2012).</p>
<p>38        Barkovich, A. J. <em>Pediatric neuroimaging</em>.  (Lippincott Williams &amp; Wilkins, 2005).</p>
<p>39        Tymofiyeva, O., Hess, C. P., Xu, D. &amp; Barkovich, A. J. Structural MRI connectome in development: challenges of the changing brain. <em>Brit J Radiol</em> <strong>87</strong>, doi:ARTN 20140086</p>
<p>10.1259/bjr.20140086 (2014).</p>
<p>40        Kelly, C., Biswal, B. B., Craddock, C., Castellanos, F. X. &amp; Milham, M. P. Characterizing variation in the functional connectome: promise and pitfalls. <em>Trends Cogn Sci</em> <strong>16</strong>, 181-188, doi:10.1016/j.tics.2012.02.001 (2012).</p>
<p>41        Bullmore, E. &amp; Sporns, O. Complex brain networks: graph theoretical analysis of structural and functional systems (vol 10, pg 186, 2009). <em>Nature Reviews Neuroscience</em> <strong>10</strong>, doi:10.1038/nrn2618 (2009).</p>
<p>42        Horwitz, B., Mcintosh, A. R., Haxby, J. V. &amp; Grady, C. L. Network Analysis of Brain Cognitive Function Using Metabolic and Blood-Flow Data. <em>Behav Brain Res</em> <strong>66</strong>, 187-193, doi:Doi 10.1016/0166-4328(94)00139-7 (1995).</p>
<p>43        Kasthuri, N. &amp; Lichtman, J. W. Neurocartography. <em>Neuropsychopharmacology</em> <strong>35</strong>, 342-343, doi:10.1038/npp.2009.138 (2010).</p>
<p>44        Lynall, M. E. <em>et al.</em> Functional Connectivity and Brain Networks in Schizophrenia. <em>Journal of Neuroscience</em> <strong>30</strong>, 9477-9487, doi:10.1523/Jneurosci.0333-10.2010 (2010).</p>
<p>45        Meunier, D., Achard, S., Morcom, A. &amp; Bullmore, E. T. Age-related changes in modular organization of human brain functional networks. <em>Neuroimage</em> <strong>44</strong>, 715-723, doi:10.1016/j.neuroimage.2008.09.062 (2009).</p>

      
    </div>
    
    
    

<div>
  
    <div>
    
        <div style="text-align:center;color: #ccc;font-size:14px;">-------------End-------------</div>
    
</div>
  
</div>

    

    

    

    <footer class="post-footer">
      
        <div class="post-tags">
          
            <a href="/tags/Neuroscience/" rel="tag"><I class="fa fa-tag"></i> Neuroscience</a>
          
        </div>
      

      
      
        <div class="post-widgets">
        

        

        
          
          <div id="needsharebutton-postbottom">
            <span class="btn">
              <i class="fa fa-share-alt" aria-hidden="true"></i>
            </span>
          </div>
        
        </div>
      
      

      
        <div class="post-nav">
          <div class="post-nav-next post-nav-item">
            
              <a href="/2018/09/12/hello-world/" rel="next" title="Hello World">
                <i class="fa fa-chevron-left"></i> Hello World
              </a>
            
          </div>

          <span class="post-nav-divider"></span>

          <div class="post-nav-prev post-nav-item">
            
              <a href="/2018/09/26/mojave/" rel="prev" title="MacOS 10.14 "Mojave" Wallpaper">
                MacOS 10.14 "Mojave" Wallpaper <i class="fa fa-chevron-right"></i>
              </a>
            
          </div>
        </div>
      

      
      
    </footer>
  </div>
  
  
  
  </article>



    <div class="post-spread">
      
    </div>
  </div>


          </div>
          


          

  
    <div class="comments" id="comments">
    </div>
  



        </div>
        
          
  
  <div class="sidebar-toggle">
    <div class="sidebar-toggle-line-wrap">
      <span class="sidebar-toggle-line sidebar-toggle-line-first"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-middle"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-last"></span>
    </div>
  </div>

  <aside id="sidebar" class="sidebar">
    
    <div class="sidebar-inner">

      

      
        <ul class="sidebar-nav motion-element">
          <li class="sidebar-nav-toc sidebar-nav-active" data-target="post-toc-wrap">
            Table of Contents
          </li>
          <li class="sidebar-nav-overview" data-target="site-overview-wrap">
            Overview
          </li>
        </ul>
      

      <section class="site-overview-wrap sidebar-panel">
        <div class="site-overview">
          <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
            
              <p class="site-author-name" itemprop="name">Jason Zheng</p>
              <p class="site-description motion-element" itemprop="description">Hi there!</p>
          </div>

          <nav class="site-state motion-element">

            
              <div class="site-state-item site-state-posts">
              
                <a href="/archives/">
              
                  <span class="site-state-item-count">5</span>
                  <span class="site-state-item-name">posts</span>
                </a>
              </div>
            

            

            
              
              
              <div class="site-state-item site-state-tags">
                <a href="/tags/index.html">
                  <span class="site-state-item-count">4</span>
                  <span class="site-state-item-name">tags</span>
                </a>
              </div>
            

          </nav>

          
            <div class="feed-link motion-element">
              <a href="/atom.xml" rel="alternate">
                <i class="fa fa-rss"></i>
                RSS
              </a>
            </div>
          

          
            <div class="links-of-author motion-element">
                
                  <span class="links-of-author-item">
                    <a href="mailto:lacroqueta.jason@gmail.com" target="_blank" title="E-Mail">
                      
                        <i class="fa fa-fw fa-envelope"></i></a>
                  </span>
                
                  <span class="links-of-author-item">
                    <a href="https://t.me/LaCroqueta" target="_blank" title="Telegram">
                      
                        <i class="fa fa-fw fa-telegram"></i></a>
                  </span>
                
                  <span class="links-of-author-item">
                    <a href="https://twitter.com/LaaaCroqueta" target="_blank" title="Twitter">
                      
                        <i class="fa fa-fw fa-twitter"></i></a>
                  </span>
                
                  <span class="links-of-author-item">
                    <a href="https://www.instagram.com/lacroquetajason" target="_blank" title="Instagram">
                      
                        <i class="fa fa-fw fa-instagram"></i></a>
                  </span>
                
                  <span class="links-of-author-item">
                    <a href="https://github.com/LaaaCroqueta" target="_blank" title="GitHub">
                      
                        <i class="fa fa-fw fa-github"></i></a>
                  </span>
                
            </div>
          

          
          

          
          

          

        </div>
      </section>

      
      <!--noindex-->
        <section class="post-toc-wrap motion-element sidebar-panel sidebar-panel-active">
          <div class="post-toc">

            
              
            

            
              <div class="post-toc-content"><ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Abstract"><span class="nav-number">1.</span> <span class="nav-text">Abstract</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Introduction"><span class="nav-number">2.</span> <span class="nav-text">Introduction</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Scales-and-Levels"><span class="nav-number">3.</span> <span class="nav-text">Scales and Levels</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Microscale"><span class="nav-number">3.1.</span> <span class="nav-text">Microscale</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Macroscale"><span class="nav-number">3.2.</span> <span class="nav-text">Macroscale</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Mesoscale"><span class="nav-number">3.3.</span> <span class="nav-text">Mesoscale</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Techs-and-Methods"><span class="nav-number">4.</span> <span class="nav-text">Techs and Methods</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Challenges"><span class="nav-number">5.</span> <span class="nav-text">Challenges</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Complexity"><span class="nav-number">5.1.</span> <span class="nav-text">Complexity</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Individual-Variability"><span class="nav-number">5.2.</span> <span class="nav-text">Individual Variability</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Parcellations-and-Scales"><span class="nav-number">5.3.</span> <span class="nav-text">Parcellations and Scales</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Techs"><span class="nav-number">5.4.</span> <span class="nav-text">Techs</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Outcome"><span class="nav-number">6.</span> <span class="nav-text">Outcome</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Reference"><span class="nav-number">7.</span> <span class="nav-text">Reference</span></a></li></ol></div>
            

          </div>
        </section>
      <!--/noindex-->
      

      

    </div>
  </aside>


        
      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="footer-inner">
        <div class="copyright">&copy; <span itemprop="copyrightYear">2018</span>
  <span class="with-love">
    <i class="fa fa-user"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Jason Zheng</span>

  
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-area-chart"></i>
    </span>
    
      <span class="post-meta-item-text">Site words total count&#58;</span>
    
    <span title="Site words total count">6.2k</span>
  
</div>









        







        
      </div>
    </footer>

    
      <div class="back-to-top">
        <i class="fa fa-arrow-up"></i>
        
      </div>
    

    

  </div>

  

<script type="text/javascript">
  if (Object.prototype.toString.call(window.Promise) !== '[object Function]') {
    window.Promise = null;
  }
</script>









  


  











  
  
    <script type="text/javascript" src="/lib/jquery/index.js?v=2.1.3"></script>
  

  
  
    <script type="text/javascript" src="/lib/fastclick/lib/fastclick.min.js?v=1.0.6"></script>
  

  
  
    <script type="text/javascript" src="/lib/jquery_lazyload/jquery.lazyload.js?v=1.9.7"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/fancybox/source/jquery.fancybox.pack.js?v=2.1.5"></script>
  

  
  
    <script type="text/javascript" src="/lib/canvas-nest/canvas-nest.min.js"></script>
  


  


  <script type="text/javascript" src="/js/src/utils.js?v=5.1.4"></script>

  <script type="text/javascript" src="/js/src/motion.js?v=5.1.4"></script>



  
  


  <script type="text/javascript" src="/js/src/affix.js?v=5.1.4"></script>

  <script type="text/javascript" src="/js/src/schemes/pisces.js?v=5.1.4"></script>



  
  <script type="text/javascript" src="/js/src/scrollspy.js?v=5.1.4"></script>
<script type="text/javascript" src="/js/src/post-details.js?v=5.1.4"></script>



  


  <script type="text/javascript" src="/js/src/bootstrap.js?v=5.1.4"></script>



  


  




	





  





  










  <script src="//cdn1.lncld.net/static/js/3.0.4/av-min.js"></script>
  <script src="//unpkg.com/valine/dist/Valine.min.js"></script>
  
  <script type="text/javascript">
    var GUEST = ['nick','mail','link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item=>{
      return GUEST.indexOf(item)>-1;
    });
    new Valine({
        el: '#comments' ,
        verify: false,
        notify: false,
        appId: 'IKgiGEsSMAvEOCWHdbybKUKg-gzGzoHsz',
        appKey: 'pWivSJh5guebh6UQ7SvADJrh',
        placeholder: 'Anything, anything.',
        avatar:'hide',
        guest_info:guest,
        pageSize:'10' || 10,
    });
  </script>



  





  

  

  

  
  
  
  <link rel="stylesheet" href="/lib/needsharebutton/needsharebutton.css">

  
  
  <script src="/lib/needsharebutton/needsharebutton.js"></script>

  <script>
    
      pbOptions = {};
      
          pbOptions.iconStyle = "defalut";
      
          pbOptions.boxForm = "horizontal";
      
          pbOptions.position = "bottomMiddle";
      
          pbOptions.networks = "Weibo,Wechat,Douban,QQZone,Twitter,Facebook,Evernote,Reddit,Pinterest,Tumblr,GooglePlus,Linkedin";
      
      new needShareButton('#needsharebutton-postbottom', pbOptions);
    
    
  </script>

  

  

  

  

</body>
</html>
